index.js

```diff
@@ -18,4 +18,6 @@ aws.Request.prototype.promise = function() {
   });
 };
 
+aws.Promise = undefined;
```

Comment:
Well, I didn't want to load `promise` library unless necessary. Maybe that's overkill, but I figured it's one less dependency to worry about if bring my own library.